src/Listener/UserCoverRelationship.php

```diff
@@ -37,23 +46,26 @@ public function __construct(Paths $paths)
      */
     public function __invoke(UserSerializer $serializer, User $user): array
     {
+        $local = $this->coversDir instanceof Local;
```

Comment:
I don't believe we need to check if the driver is local, we can directly call `$this->coversDir->url('..')` since the local driver implements the Cloud contract. Also, I don't believe there is an `Illuminate\Contracts\Filesystem\Local` contract 🤔 

This isminor though, would you like to merge this as is and I can make changes another time?

Reply:
🤦 you're absolutely correct. Pushed that change just now as I still had this locally 👍